### DISCUSSION TITLE
Disable JWT verification for database function

### DISCUSSION THREAD

**Discussion by pbuyle** [supabase/supabase#12655] [Questions]
Hi,

is there a way to disable JWT verification for database function? Same as what the `--no-verify-jwt` does when creating an edge function, but for a database on. Allowing a POST request with no `apikey` and `Authorization` header to go execute the function.

**Comment by TheCoderJT** [comment]
I would like this too, stripe doesn't return JWT token, so you will find 401 errors and so on. I tried to manually disable it in details but then it keeps getting automatically enabled.

**Reply by adavid00** [reply]
same problem with stripe here

**Comment by Studiosbyacpk** [comment]
Same problem as TheCoderJT!
any admin or supercoder with a solution?

I have a couple of edge functions in Supabase database for my project. I want these to have disabled JWT Verification. in each of the functions config.toml file I have written a code simply like this:

verify_jwt = false

however, once in a while ( I think everytime I update the database) the JWT verification is enabled. since I have alot of edge functions it take alot of time to disable all of these plus I have problems in the mean time with edge function not working properly due to the fact that JWT verification has been enabled misstakenly.

**Comment by GaryAustin1** [comment]
The last two posts here seem to be on edge functions and having issues with that.  You might want to start a new topic or issue as the issue here is wanting to do DATABASE functions without a JWT which his not possible directly.
